conductor-client/src/main/kotlin/com/openlattice/postgres/external/Schemas.kt

```diff
@@ -0,0 +1,27 @@
+package com.openlattice.postgres.external
+
+/**
+ * @author Drew Bailey (drew@openlattice.com)
+ *
+ * Schemas:
+ * - "entitysets"
+ *      schema in org_* database where views are assembled
+ * - "transporter"
+ *      schema in org_* database where foreign et tables are accessible
+ * - "ol"
+ *      schema in atlas transporter database where production tables are accessible
+ */
+enum class Schemas(val label: String) {
+    INTEGRATIONS_SCHEMA("integrations"),
+    OPENLATTICE_SCHEMA("openlattice"),
+    TRANSPORTED_VIEWS_SCHEMA("ol"),
+    PUBLIC_SCHEMA("public"),
+    TRANSPORTER_SCHEMA("transporter"),
+    ASSEMBLED_ENTITY_SETS("entitysets"),
+    ENTERPRISE_FDW_SCHEMA("ol"),
```

Comment:
`TRANSPORTED_VIEWS_SCHEMA` and `ENTERPRISE_FDW_SCHEMA` are both `ol`. Is it possible to have multiple `ol` schemas?

Reply:
Its not possible to have multiple of the same schema in one database. This is effectively two enum values for one schema name. Probably ENTERPRISE_FDW_SCHEMA should be replaced by TRANSPORTED_VIEWS_SCHEMA

Reply:
Looks like TRANSPORTED_VIEWS_SCHEMA was unused